### DISCUSSION TITLE
Cura settings for 2.0A Duo?

### DISCUSSION THREAD

**Discussion by M1-Ace** [luc-github/Repetier-Firmware-0.92#398] [Q&A]
I just installed the newest repetier firmware onto my 2.0A Duo, and I can't figure out what settings I need in cura (build volume, extruder offset, etc.).

**Comment by luc-github** [comment]
which newest repetier FW ? there was no update for years ^_^

**Reply by luc-github** [reply]
you do not need to set extruder offset in cura but you need to do your calibration according your printer and update EEPROM setting

**Reply by M1-Ace** [reply]
1. Im new to dual extruders, but wuldn't I need to put in the offset? There's at least an inch in between both extruders

2. Is there an easy way that I could view the eeprom settings?

**Reply by luc-github** [reply]
you can see in your printer lcd menu, or use repetier host tonedit EEPROM
you do not need offset in cura because when you select T0 T1 the FW itself move extruder to right position, that is why it is important calibration is done like explained in readme

**Reply by M1-Ace** [reply]
I can see that it is calling t0 and t1, but it always seems to be using the same extruder, which causes the second one to try to print off of the build plate. I've tried manually adding offsets but those don't seem to have any effect.

**Reply by luc-github** [reply]
it means you did not setup cura properly to support 2 extruders - for that you must read cura documentation, if you always estract one extruder issue - is in cura not in FW, again you do not need to define extruders offset in cura as they are useless with repetier which do the job to put extruder in right position, but it is slicer to select extruder to extrude

**Reply by luc-github** [reply]
whatch this it may help to understand dual extrusion : https://www.youtube.com/watch?v=xRtvbICRh1w
and 
https://www.youtube.com/watch?v=ZPnU-w40j6k

One common mistake is only to heat one extruder and not heat the second

**Reply by luc-github** [reply]
for offset extruder calibration - I used https://www.thingiverse.com/thing:21773
once your cone is clean it means your offset are ok - it is trial error process as every dual extruder has different values
